Packs/FeedProofPoint/Integrations/FeedProofPoint/FeedProofPoint.yml

```diff
@@ -0,0 +1,122 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: ProofpointFeed
+  version: -1
+configuration:
+- display: Fetch indicators
```

Comment:
Also, look at the document linked [here](https://demisto.slack.com/archives/CRT0G4Y82/p1580053983004500) for how you should organize your integration's parameters.